tests/test_rpc_methods_availability.py

```diff
@@ -0,0 +1,27 @@
+import delayed_assert
+from substrateinterface import SubstrateInterface
+
+# "system_syncState" can be used for test, not available on Acala, Astar
+
+rpc_methods = [
+    "state_call", "state_getStorage" "state_getStorageSize", "state_subscribeStorage", "state_getKeys",
+    "state_getKeysPaged", "state_getMetadata", "state_subscribeRuntimeVersion",
+    "system_chain", "system_accountNextIndex", "system_properties"
+    "chain_getBlock", "chain_getBlockHash", "chain_getHeader", "chain_getFinalizedHead",
+    "chain_getRuntimeVersion", "chain_subscribeRuntimeVersion",
+    "childstate_getStorage", "payment_queryInfo",
```

Comment:
payment_queryInfo would be better to move in separate flow, in order to set param - "feeViaRuntimeCall": true
```suggestion
    "childstate_getStorage",
```